sql/modules/Roles.sql

```diff
@@ -95,8 +100,11 @@ CREATE OR REPLACE FUNCTION lsmb__grant_menu
 RETURNS BOOL
 LANGUAGE PLPGSQL SECURITY INVOKER AS
 $$
+DECLARE
+  t_in_role text;
 BEGIN
-   PERFORM * FROM pg_roles WHERE rolname = lsmb__role(in_role);
+   SELECT lsmb__role(in_role) INTO t_in_role;
```

Comment:
Could you repeat the comment you added above, here too?

Reply:
Done